test-requirements.txt

```
@@ -0,0 +1 @@
+WebTest==2.0.32
```

```suggestion
webtest==2.0.32
```

We have them all lowercase in the main requirements

---

I hadn't realised pip wasn't case-sensitive.